static/js/components/Navbar.js

```diff
@@ -27,12 +26,11 @@ export default class Navbar extends React.Component {
     addProgramEnrollment:        (programId: number) => void,
     children?:                   React$Element<*>[],
     currentProgramEnrollment:    ProgramEnrollment,
-    dashboard:                   DashboardState,
     empty:                       boolean,
     enrollDialogError:           ?string,
     enrollDialogVisibility:      boolean,
     enrollSelectedProgram:       ?number,
-    programs:                    ProgramEnrollmentsState,
+    programs:                    ProgramEnrollments,
```

Comment:
what do you think of changing the variable names and type names from `ProgramEnrollments` (et al) to `AvailablePrograms`? we still have this awkwardness with 'Enrollments'. `programs: ProgramEnrollments`; `programs: PROGRAM_ENROLLMENTS`; etc. maybe `AvailablePrograms` isnt perfect, but it's accurate in that the list is supposed to represent every program available in Micromasters. 

this would change the `ProgramEnrollmentsState` like so:

```javascript
export type AvailableProgramsState = {
  availablePrograms: AvailablePrograms,
  getStatus?: string,
  getErrorInfo?: APIErrorInfo,
  postStatus?: string,
};
```